recipes/multirtc/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "multirtc" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
```

Comment:
We only need to set `python_min` if the min version is higher than conda-forge's min. 

```suggestion
```